wbooks/src/app/screens/Login/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { ImageBackground, Text, TouchableOpacity } from 'react-native';
+import Form from '@screens/Login/components/Form';
+import bcInicio from '@assets/General/bc_inicio.png';
+
+import styles from './styles';
+
+const Login = () => {
+  return (
+    <ImageBackground source={bcInicio} style={styles.container}>
+      <Form title="Email" />
+      <Form title="Password" isSecure={true} />
+      <TouchableOpacity style={styles.loginBtn}>
```

Comment:
touchables should always have an `onPress` prop, even. if you must write a empty function like `() =>{}`

Reply:
Ok, the linter doesn't admit empty arrow function, so I put a console.log